sunpy/io/_jp2.py

```diff
@@ -1,22 +1,43 @@
 """
 This module provides a JPEG 2000 file reader for internal use.
 
-.. warning::
-
-    ``sunpy.io.jp2`` is deprecated, and will be removed in sunpy 4.1. This is
-    because it was designed for internal use only.
+It was developed to read JPEG 2000 files created by the Helioviewer Project
+and not all JPEG 2000 files are supported.
 """
 import os
-from xml.etree import ElementTree as ET
 
+import lxml.etree as ET
 import numpy as np
 
 from sunpy.io.header import FileHeader
-from sunpy.util.io import HDPair, string_is_float
-from sunpy.util.xml import xml_to_dict
+from sunpy.util.io import HDPair
 
 __all__ = ['read', 'get_header', 'write']
 
+# TODO: Do we need to do this to the values?
+def _sanative_value(value):
```

Comment:
These are not directly tested but covered by the jp2 tests, which I can expand to ensure we are doing things correctly?